Rewrite YQL CREATE

Pseudolukian opened the pull request:
### Changelog category
* Documentation (changelog entry is not required)

Pseudolukian committed:
Fix includes

Pseudolukian committed:
Fix concepts err

fomichev3000 commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/secondary_index.md:
```diff
@@ -0,0 +1,69 @@
+# INDEX
+
+Конструкция `INDEX` используется для определения {% if concept_secondary_index %}[вторичного индекса]({{ concept_secondary_index }}){% else %}вторичного индекса{% endif %} {% if backend_name == "YDB" %}для [строковых](../../../../concepts/datamodel/table.md#row-oriented-tables) и [колоночных](../../../../concepts/datamodel/table.md#column-oriented-tables) таблиц{% else %}на таблице{% endif %}:
```
Для колоночных вторичного индекса нет.

Pseudolukian replied:
Поправил

fomichev3000 commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/family.md:
```diff
@@ -0,0 +1,34 @@
+# Группы колонок (FAMILY)
+
+Колонки одной таблицы можно объединять в группы, для того чтобы задать следующие параметры:
```
family нет для колоночных таблиц

Pseudolukian replied:
Поправил

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Intermediate changes

Commit:
TS_PACKAGE by glob

Добавляем возможность описывать список файлов для TS_PACKAGE с помощью GLOB'ов
b10e0775708047bb73326dbce495e367903c3323

robot-piglet committed:
Intermediate changes

Commit:
DeprecatedField -> VirtualField; also support saving virtual fields
488c5618adaa789790af173bbb63fb9354af45a8

robot-piglet committed:
Intermediate changes

Commit:
YT-21993: Check compilation of a refcounted abstract class in Phoenix v2
808994a8e6d0f64eb17e639514cf2b4cc3b12f54

robot-piglet committed:
Intermediate changes

E1pp committed:
YT-21233: Split error into error and stripped_error to rid the latter of deps on global variables

What happened:
1. error contents has been split into stripped_error and error. stripped_error contains the error itself (with attributes for now) and macros; error contains stripped_error and some extensions, namely, functions to get fiberId, hostname and traceid/spanid and all functions used to (de-)serialize error. This means that you cannot print error if you only include stripped_error, therefore you are likely to still require the entire error.h at the moment.
2. Mechanic for gathering origin attributes has been moved to newly created library/cpp/yt/error thus having no dependency on fibers, net or tracing. stripped_error uses these attributes as extendable semi-erased (meaning, you still would have to add a field and recompile the entire thing, but you don't have to introduce an extra dependency) storage for a bunch of attributes
3. Parsing of said attributes is done in error file (and not stripped_error).

P.S. So far the plan is to eventually move stripped_error (once dependency on core/ytree/attributes is eliminated) without any actual change to dependency graph of anything outside of core (e.g. you would still have to include misc/error.h to use it). Next step would be re-teaching the error how to print, which would move some more methods from core to the standalone module. After that one could finally depend on the error itself and not the entire core.

Annotations: [nodiff:caesar]
66615172181355821241d2e5f8e4a0f15e0ea791

robot-piglet committed:
Update contrib/restricted/boost/program_options to 1.86.0
5b0ca73a4ff83de015d9402c61d3eba99038ad97

robot-piglet committed:
Intermediate changes

ubyte committed:
check the lifetime bounds of stream adapters
f4e1f5605822d2c05fd582aeb7104301b7ca73c7

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/family.md:
```diff
@@ -0,0 +1,36 @@
+# Группы колонок
+
+{% include [not_allow_for_olap](../../../../_includes/not_allow_for_olap_note.md) %}
```
```suggestion
{% if oss == true and backend_name == "YDB" %}

{% include [not_allow_for_olap](../../../../_includes/not_allow_for_olap_note.md) %}

{% endif %}
```

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/index.md:
```diff
@@ -0,0 +1,167 @@
+    WITH ( key = value, ... )
+{% endif %}
+
+{% if backend_name == "YDB" %}
```
```suggestion
{% if oss == true and backend_name == "YDB" %}
```

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/index.md:
```diff
@@ -0,0 +1,167 @@
+  columns 
+  ...
+)
+
```
```suggestion
```

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/index.md:
````diff
@@ -0,0 +1,167 @@
+)
+```
+По умолчанию, если параметр `STORE` не указан, создается строковая таблица.
+
+{% endif %}
````
```suggestion
```

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/index.md:
````diff
@@ -0,0 +1,160 @@
+  STORE = COLUMN -- Default value ROW
+)
+```
+По умолчанию, если параметр `STORE` не указан, создается строковая таблица.
````
```suggestion
По умолчанию, если параметр `STORE` не указан, создается строковая таблица.

{% endif %}
```

blinkov commented on ydb/docs/ru/core/yql/reference/yql-core/syntax/create_table/with.md:
```diff
@@ -0,0 +1,47 @@
+
+Перечень допустимых имен параметров и их значений приведен на странице [описания таблицы {{ backend_name }}]({{ concept_table }}).
+
+Например, такой код создаст таблицу с включенным автоматическим партиционированием по размеру партиции и предпочитаемым размером каждой партиции 512 мегабайт:
```
```suggestion
Например, такой запрос создаст строковую таблицу с включенным автоматическим партиционированием по размеру партиции и предпочитаемым размером каждой партиции 512 мегабайт:
```

zaverden committed:
feat(npm): npm target with peer
861cdb3f890da6b591920637785f8de1d1267b6e

Commit:
Remove fix-elf from ymake.core.conf
bd2fb0cb2bbfb52feae17c48694ca01ef959bb8f

robot-piglet committed:
Intermediate changes

rekby committed:
Export python deepmerge library instead of mergedeep  to github.com/ydb-platform/ydb for support python2
98bbe613ba94337077da6f6bb9b519768fdef800

alexv-smirnov committed:
Import libraries 240816-1413

Commit:
ydb_topic: Unmute BasicUsage.ConflictingWrites test (#7891)

Tony-Romanov committed:
Don't queue requests for workers if their capacity is empty. (#7979)

mxkovalev committed:
YQL: Fix prompt in error message (#7888)

UgnineSirdis committed:
Interconnect: allow using not only RSA keys. Set max supported TLS version to 1.3 (#7954)

nshestakov committed:
Add tests for pqv1 (#7669)

gridnevvvit committed:
allow to configure min alloc size (#7951)

pashandor789 committed:
[CBO] Column shards tests added (#7972)

azevaykin committed:
Statistics: Send TEvAnalyzeTable to shards (#7880)

ivanmorozov333 committed:
refactoring for ranges control (#7758)

gridnevvvit committed:
add stability test to oss (#7992)

kungasc committed:
Report QueryExecution memory stats (#7936)

robot-piglet committed:
Intermediate changes

Commit:
YT-22350: Optimize conditional discount and disk quota storage
aa0dd98e9a2e4c54f4e65bd37eb75541ce8fa219

resetius committed:
Tune decimal precision for q66 (#7998)

resetius committed:
Numerics in beetween (#8004)

snaury committed:
New granular timecast protocol for mediator (#7986)

alexvru committed:
Support forbidden certificate algorithms list in IC (#7974)

Commit:
Add contrib/ydb/docs into piglet (broken docs build in github)

Как оказалось у нас не собирается сборка доков на гитхабе

Явно добавляю зависимость на ydb/docs для импорта
4111a26c18b0848d55ec83f88e2f1f7dab7cacc1

E1pp committed:
Remove unjustified noexcept specifier
1ea40a2a3daf0f27f05cde14504f261d2cc902c6

Tony-Romanov committed:
Don't wait in scheduler queue if all workers out. (#8002)

azevaykin committed:
TSerializedCellVec operator bool (#7981)

Commit:
Fix LSP in future-inl.h
71ff0471827f8bde6feb9549ead39faefb7f9b0f

lifthelm committed:
YDB FQ: Exclude DATABASE_NAME from mandatory for oracle (#7593)

siarheivesialou committed:
Finally fix use after move in http_proxy correctly (#7991)

dcherednik committed:
Check uuid in pk replication works (#7950)

GrigoriyPA committed:
YQ-3154 improve error in s3 applicator actor (#7976)

pavelvelikhov committed:
Fixed a subtle bug in removing aliases (#8003)

Commit:
Allow system sampling in DQ (#7836)

alexv-smirnov committed:
Library import 240819-0942 (#7994)

Co-authored-by: robot-piglet <robot-piglet@yandex-team.com>
Co-authored-by: hiddenpath <hiddenpath@yandex-team.com>
Co-authored-by: bulatman <bulatman@yandex-team.com>
Co-authored-by: robot-contrib <robot-contrib@yandex-team.com>
Co-authored-by: osidorkin <osidorkin@yandex-team.com>
Co-authored-by: akhropov <akhropov@yandex-team.com>
Co-authored-by: akozhikhov <akozhikhov@yandex-team.com>
Co-authored-by: orlovorlov <orlovorlov@yandex-team.com>
Co-authored-by: babenko <babenko@yandex-team.com>
Co-authored-by: shadchin <shadchin@yandex-team.com>
Co-authored-by: dmasloff <dmasloff@yandex-team.com>
Co-authored-by: aleksei-le <aleksei-le@yandex-team.com>
Co-authored-by: coteeq <coteeq@yandex-team.com>
Co-authored-by: dimdim11 <dimdim11@yandex-team.com>
Co-authored-by: robot-ya-builder <robot-ya-builder@yandex-team.com>
Co-authored-by: iaz1607 <iaz1607@yandex-team.com>

alexv-smirnov committed:
Import libraries 240819-1422

dimdim1177 committed:
Fix ASM as last project language

Fix ASM as last project language
bce309873abd11037932fbd4a53a8ad8f23b881f

dimdim1177 committed:
Fix ASM as last project language 2

Fix ASM as last project language 2
55356e7dfcf81cebdedc53a7ef5f58f634b18cb4

maximyurchuk committed:
Add codeowner for arcadia->github sync (#8016)

stanislav-shchetinin committed:
Added new versions of RetryQuery (#7777)

blinkov committed:
[docs] fix a link path in concat.md (#8025)

maximyurchuk committed:
Remove ram requirements in sqs for sanitizers (#8022)

maximyurchuk committed:
Remove fq/generic from asan precommit checks (too long, and not profitable) (#8026)

shadchin committed:
Prepare to switch on platform_dispatchers (part 1)
664a0b8555cd24bd063b0fa871e3e5693e856e7d

robot-piglet committed:
Intermediate changes

alexander-somov committed:
Fix LIST_PROTO dependency on incorrect properties passing between main and additional outputs
bb4a52dc41ed160f50f9dece539dbd788be97430

alexv-smirnov committed:
Import libraries 240819-2038

robot-piglet committed:
Intermediate changes

igormunkin committed:
Introduce computation node for BlockMapJoinCore (LeftOnly and LeftSemi) (#7947)

zaverden committed:
feat(conf): disable FROM_NPM_LOCKFILES logic
e4ed06d919d0a10c836deb466f29723dfa01b63a

Commit:
support `next experimental-compile` command

Добавляем возможность использовать в `next` экспериментальную функциональность построения проекта без рендеринга статики, отложив рендеринг на момент запуска проекта на конкретной среде.
Это позволяет проектам, требующим разных параметров рендеринга для статики в зависимости от среды пережить ограничения автосборки, требующей независимости результата от конкретной среды, на которой выполняется сборка.
40cab21c170e2c68291be3e695045541b3984e48

Commit:
Automatic release build for ymake, os_ymake

From hash: [f8fbe3627553dc1353ece3c098bb5df2ab788b92](https://a.yandex-team.ru/arcadia/commit/f8fbe3627553dc1353ece3c098bb5df2ab788b92)
From revision: [14658523](https://a.yandex-team.ru/arcadia/commit/rXXXXXX)
[CI flow](https://a.yandex-team.ru/projects/ya_make/ci/releases/flow?dir=devtools%2Fya&id=release-ymake&version=221)
Flow triggered by user: [robot-ci](https://staff.yandex-team.ru/robot-ci)

Update tools: ymake, os_ymake
b85187767999019eefcb87a0e2bd45a8784fc4e7

Pseudolukian committed:
Rewrite YQL ALTER TABLE (#6340)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

Commit:
YDB FQ: docs for Microsoft SQL Server as an external data source (#7786)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

Pseudolukian committed:
Refactoring YDB concepts (#5580)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

SloNN committed:
integrations en version (#7912)

Co-authored-by: Alex Dmitriev <slonnn@yandex-team.ru>
Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

maximyurchuk committed:
Remove providers/generic and core/external_sources from asan precommit checks (too long, and not profitable) (#8034)

achulkov2 committed:
Introduce enable_optimize_read_in_order in CHYT

This PR introduces a new mode in CHYT that aims to mimic the behaviour of [optimize_read_in_order](https://clickhouse.com/docs/en/sql-reference/statements/select/order-by#optimization-of-data-reading) in native ClickHouse. This PR implements #451, albeit using a different approach.
It works by using a sorted pool to produce jobs of exponentially increasing/decreasing sizes, which turn into subqueries that are processed combined sequentially without additional sorting steps. This should allow queries described in #451 to execute without reading the whole table.
Notable modifications with generic impact:
* Support for exponentially increasing job sizes in new sorted pool and related classes.
* Chunk slice fetcher is now used when creating a sorted pool in CHYT.
* New column in query log table exporter.

---
1f9ea932d21d4b328b4be976eb39cdfbc631dffd

Pull Request resolved: https://github.com/ytsaurus/ytsaurus/pull/757

nepal committed:
Do not use -Oz on windows (#7960)

robot-piglet committed:
Update contrib/restricted/boost/assert to 1.86.0
2e37a39c10bebc34441ebd2c99a27d4cae188a3c

robot-piglet committed:
Update contrib/restricted/boost/smart_ptr to 1.86.0
a2c030e4658166d4aaf432632b50891e6ce35c7b

nepal committed:
Fix FilterPushdownEnableMultiusage on dependent predicates (#8033)

vitstn committed:
Support of OPERATOR CLASS for PG extensions (#8031)

robot-piglet committed:
Update contrib/restricted/boost/math to 1.86.0
26ffe6aa485d878318cbc7d5a6d850e6c1121dd1

ArtemTrofimushkin committed:
YDBOPS-9806: add note about host restart delay in docs (#7985)

siarheivesialou committed:
Use 'mock_cloud' if empty string is passed as cloud id to TFolderServiceAdapterMock (#8037)

evanevanevanevannnn committed:
trunc windows build fix (#8014)

GrigoriyPA committed:
YQ-3568 fix error sent large get operation (#8035)

maximyurchuk committed:
Remove ram requirements in fq (speed up asan checks) (#8023)

shmel1k committed:
[misc] add 24-2-7 to downloads folder (#8038)

Commit:
Revert "YT-21306: Add EnumHasDefaultValue"

This reverts commit d9b67f1778da2d15dd94f7285afe4e3490a233ab, reversing
changes made to 461a09e0c18bd14cef7df8060e7f9537e3ad74b5.
92cdaf4185661b7058f6a30d5a532ad40b725345

robot-piglet committed:
Update contrib/restricted/boost/graph to 1.86.0
29609809abb032ea449cf4b749abf84c9b9c4c1d

azevaykin committed:
Statistics: test for SA reboot (#8012)

swalrus1 committed:
Add a scenario test for altering tiering config (#7636)

fexolm committed:
Add olap_workload to stability_tests (#8047)

Commit:
YDB FQ: unmute YDB tests (#8053)

lll-phill-lll committed:
disabled block engine for spilling (#7778)

ijon committed:
auditlog: refactor ydb/core/audit library interface (#8027)

- separate library interface into client and service sides: `audit_log.h` and `audit_log_service.h`
- move all implementation into the single `audit_log_impl.cpp`
- change `CreateAuditWriter`'s `logBackends` arg to rvalue ref

KIKIMR-21774

Tony-Romanov committed:
Check status of TEvPutResult (#7579)

Enjection committed:
Decouple incremental backup from async replication (#7667)

alexv-smirnov committed:
Import libraries 240820-1626

naspirato committed:
Update TESTOWNERS (#8045)

va-kuznecov committed:
Add more introspection into PDisk's ChunkWrite and LogWrite (#7941)

Co-authored-by: Vlad Kuznecov <va-kuznecov@nebius.com>

robot-piglet committed:
Intermediate changes

dimdim1177 committed:
New CMake generator style fixes

Export style fixes
92d1ac710c97841b0e2a51ae1404b343ce3895a4

VestniK committed:
Update ymake python3 version
4873f108e99caf99c62947767f40914bedff954e

Commit:
Fix latest import problems (LLVM include) (#8021)

zverevgeny committed:
Simplify tpc-ds q34, q73 a bit to help an optimizer (#7969)

Commit:
Add TPriorityQueue::PopValue
ce37597283b1508bdba021056c149fe16feb1c69

alexv-smirnov committed:
Import libraries 240820-2236

robot-piglet committed:
Intermediate changes

ijon committed:
auditlog: add logins (#7546)

Add audit logging for login operation.

KIKIMR-21774

alex268 committed:
Docs/java example update (#7704)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Update contrib/restricted/boost/move to 1.86.0
b1ffaaff544bd1ab1af72314e524d49221a5f67e

Commit:
YDB FQ: Update github.com/ydb-platform/fq-connector-go to 0.5.3-rc.2 (#8059)

tatakir committed:
Save peers and direct peers to dm cache
3ca03f9795edc451ae7c7513f2869c3d0cfb0ff2

nshestakov committed:
Add test create datastream with disabled autopartitioning (#8072)

gridnevvvit committed:
fixes applied (#8044)

vitstn committed:
Exposed extensions via pg_extension table (#8062)

jepett0 committed:
Enable DROP VIEW from a folder, bugfix (#8066)

MrLolthe1st committed:
Fix native file test (#7935)

StekPerepolnen committed:
ExecuteData set transaction mode always (#7982)

Co-authored-by: Ivan Sukhov <evanevannnn@ydb.tech>

Commit:
Add another part SDC contrib to allow NO_LINT

Еще одна пачка проектных контрибов.
53591f2d7360aa45b12fa2215945c9fc3569df19

robot-piglet committed:
Intermediate changes

jepett0 committed:
Restore indexes from backup with the original partitioning (#7589)

robot-piglet committed:
Intermediate changes

azevaykin committed:
Statistics: ANALYZE tests for SA reboot (#8077)

achulkov2 committed:
Add dynamic table log writer

Adds a new `dynamic_table` log writer, which writes log events to the configured sorted or ordered dynamic table.
Log events are serialized into a YSON representation, which is interpreted similarly to the `insert-rows` cli command.
It is your responsibility to create the table with a compatible schema.
See integration test for example configuration.

The writer is available for use in all YT server components.

TRIVIAL

---
bfb7bf098156a8b16a6fdbf018c54858b1479b03

Pull Request resolved: https://github.com/ytsaurus/ytsaurus/pull/631

Co-authored-by: eshcherbin <eshcherbin@yandex-team.com>

GrigoriyPA committed:
YQ kqprun enable runtime listing by default (#8096)

pixcc committed:
Add sensors to Node Broker (#8091)

rvu1024 committed:
Add examples for LAG (#8082)

robot-piglet committed:
Intermediate changes

molotkov-and committed:
OIDC proxy: Handle response with bad request from protected resource (#8105)

azevaykin committed:
Statistics: Traverse tests refactoring (#8110)

Commit:
Track job CPU utilization on Flow workers
8fc3af8a8fe764c4c60363a67733aedeb70c9226

VPolka committed:
return implicit query params type into feature flag (#8065)

robot-piglet committed:
Intermediate changes

GrigoriyPA committed:
YQ-3363 fix internal error for insert without params (#8074)

Commit:
External build system generator release 98

Update tools: yexport, os-yexport
c6e64241d2cbdcf9e95017a87c0e3959e88db79d

kruall committed:
Change activation queue to variant of queues (#8078)

Commit:
YT-21838: Introduce nested aggregate columns
6bac1e5898fd226d2152bf14df7e1a2b227895aa

Commit:
YT-21993: Allow to omit including `type-def.h` in header files that use Phoenix v2
02dce8a147c6036c91d4c69dac40e9a9993fc26d

Commit:
YT-21993: Add forgotten `override` specifiers
b77c55a40214fce1a37af17fb6d9bf7e04805eef

vitstn committed:
Fixes #8019 (#8109)

vporyadke committed:
improve failsafe on cuttting history (#6918)

Commit:
YT-21993: Allow to mix Phoenix v1 and v2 classes in the hierarchy
da8e97a3a6b4fdacb65a37420ccb2ec916ab604b

robot-piglet committed:
Update contrib/restricted/boost/bind to 1.86.0
36e5db27fff0e5e63069b8863a31054cd7e5b443

niksaveliev committed:
Fix describe consumer (#8095)

ivanmorozov333 committed:
multi heads allocations manager (#8126)

Commit:
Enable fix-elf only with _FIX_ELF flag
f1b4ab8d00e0ac2175f0ea34a0353ce2963549b7

Pseudolukian committed:
Fix YQL optional types note in YT DOC (#8057)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

SloNN committed:
pg (#7381)

Co-authored-by: Alex Dmitriev <slonnn@yandex-team.ru>
Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

zinal committed:
Moved cluster topology information to the topology overview section (#7144)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

blinkov committed:
[docs] update create_table.md (#8071)

robot-piglet committed:
Intermediate changes

dimdim1177 committed:
Fix headers only library templates

Fix headers only libraries
8d98d2115e9207d233331bf0ab2ba307d84b255c

swalrus1 committed:
Add unit-test for data eviction in tiering (#7971)

Commit:
Update github.com/ydb-platform/fq-connector-go to 0.5.5-rc.1 (#8127)

azevaykin committed:
Statistics: AnalyzeStatus should use the same sender to preserve event order (#8136)

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Update contrib/restricted/boost/predef to 1.86.0
a86177b405d83df3dba0a9c13612ef7144cdab7b

robot-piglet committed:
Update contrib/restricted/boost/container to 1.86.0
1d9f36e4b2e11f3aabe2d7350166df08c1cc72d1

snaury committed:
Show a nicer demangled name in block/unblock messages (#8138)

maximyurchuk committed:
Add docs dir into RECURSE ydb (#8092)

robot-piglet committed:
Intermediate changes

Commit:
YT-20983: Add dynamic tables versioned map reduce read
12c8b4a9cd18c32a89bb2bc6d78fd4dc0f1da7b5

dorooleg committed:
streaming grace join test (#3862)

nikitka committed:
ci: download the latest github runner on a VM start (#7987)

azevaykin committed:
Debug logging in WaiFor (#8137)

ivanmorozov333 committed:
correct portions index usage for control memory scan intervals (#8135)

swalrus1 committed:
Don't skip null values on COUNT(*) in column shards (#8024)

snaury committed:
Use granular updates for mediator timecast when available (#8090)

vitstn committed:
Update postgis functions from tests (#8133)

kruall committed:
Add UseRingQueue to ActorSystemConfig (#8122)

SammyVimes committed:
CMS editor async load fix (#8140)

Loading editor.js doesn't block the rest of the page's content loading anymore

fexolm committed:
Add scan/insert while alter test (#8094)

azevaykin committed:
Statistics: SendBaseStatsToSA return delta for schedule next event (#8119)

swalrus1 committed:
Add column shard Controller parameters to AppData (#8000)

StekPerepolnen committed:
add http-handle returning configs (#7830)

kungasc committed:
Apply old Resource Broker config (#8087)

qrort committed:
pass db path to pg_tables instead of tenant name. Resolves #7616 (#8061)

pavelvelikhov committed:
Lowered threshold for MapJoin (#8147)

robot-piglet committed:
Intermediate changes

Commit:
YT-21993: Fix duplicate type tag on derive from template
7ec29a7b370f343f910075ff9fd87ea9469d2aa0

p4vook committed:
GLOBAL OTBIVKA FIX ୧༼ಠ益ಠ༽୨

Make all "otbivka" exactly 80 symbols.

Command:
```bash
sed -i 's#^//////////////\+$#////////////////////////////////////////////////////////////////////////////////#g' $(find yt \( -name '*.cpp' -or -name '*.h' -or -name '*.proto' \) -not -path 'yt/yt/experiments/private/tabletvis/imgui/*')
```
30a0ba9f42ba1909d3cee11fe12bba14c7c8b039

Krisha11 committed:
Fix skiff usage in dq (#7864)

vitstn committed:
Fixed validation of lambda before apply (#8153)

lll-phill-lll committed:
added flamegraph to github
03edf63f51b2878f5e14d39a19a3e1aa72de5398

vporyadke committed:
actually tie per-dc-followers to dc (#8069)

Co-authored-by: Aleksandr Zalialov <zalyalov@mr-nvme-testing-01.search.yandex.net>

vporyadke committed:
treat initial vdisk status as ok (#8150)

igormunkin committed:
Refactor tests for BlockMapJoinCore computation node (#8129)

Enjection committed:
Fix config resolver in ui (#8157)

yumkam committed:
tpc ds: manually optimize join order to workaround #7565 (#8163)

StekPerepolnen committed:
query float test asan (#8161)

nikitka committed:
ci: fix logic in check-is-mergeable check (#8152)

nikitka committed:
ci: upgrade actions/checkout@v4, actions/github-script@v7, add test_retry_count input for build_and_test_ya action (#8164)

Commit:
YT-22634: Add code for total memory limit exceeded alert
2d3bed035fb10346b6295993fe1f08f714411edd

azevaykin committed:
Statistics: disable some internal events without feature flag (#8159)

Krisha11 committed:
YQLAgent, free progress memory (#7585)

Commit:
Do not use YQL_LOG during YQL plugin initialization (#6798)

mxkovalev committed:
YQL: Fuse reduces with presort settings (#8117)

alexvru committed:
Report UNKNOWN status to SysView for disks that have not checked in yet (#8144)

eivanov89 committed:
use new avx macros (#7494)

va-kuznecov committed:
PDisk LWTracing improvements (#8160)

Co-authored-by: Vlad Kuznecov <va-kuznecov@nebius.com>

Alek5andr-Kotov committed:
The value of the `WriteInflightSize` in the main partition (#8116) (#8142)

dimdim1177 committed:
Macro USE_ANNOTATION_PROCESSOR

Macro USE_ANNOTATION_PROCESSOR
670d47e64a48e59e9b190875a9729bc9e9921e15

dimdim1177 committed:
Add sem use_annotation_processor

Add sem use_annotation_processor
f5ffdd48bd9f14fc3bf52b8f4a22d0b9277305a2

maximyurchuk committed:
Do docker-compose tests in parallel (#8165)

Commit:
YT-22435: Using anonymous memory limit in tcmalloc
93d6930ce24f343ccd23c55e7264db80f2e0046b

Commit:
Update github.com/ydb-platform/fq-connector-go to 0.5.5-rc.2 (#8183)

shadchin committed:
Fix DeprecationWarning

Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.

https://docs.python.org/3/library/tarfile.html#tarfile.TarFile.extractall
79005c43af0996c47603d56b5663378c89fa9f2e

shadchin committed:
Switch on platform_dispatchers
93438876ebd102c07d52ce42cc48e5465a7277ff

maximyurchuk committed:
Revert "ci: upgrade actions/checkout@v4, actions/github-script@v7, add test_retry_count input for build_and_test_ya action" (#8185)

Commit:
New version of the tld SKIP_CHECK SKIP_REVIEW
4fbdf2c6326bb76d9e4f3f771383bc2b00a8d3e2

robot-piglet committed:
Intermediate changes

Commit:
YT-25593: Replace TString with std::string for addresses, hostnames etc
e10316e900a0dd13e98e68dacd3338f5d277c8a2

zaverden committed:
feat(conf): set requirements for build to cpu:2
c321682fac0742cd7c6fd860085140989c4729e8

pashandor789 committed:
[KQP] ANALYZE retries has been added. (#8115)

Commit:
YT-22619: Improve merge job memory tracking
38867a055f64fbf74a008e2c1dd1ef1cda73736d

robot-piglet committed:
Intermediate changes

abogutskiy committed:
Fix clang logical-op-parentheses warning

No description

---
687cd74eb85cc26a02a9e2536693ad9c9d710416

Pull Request resolved: https://github.com/ytsaurus/ytsaurus/pull/795

va-kuznecov committed:
Fix counter names in group_write load actor (#8058)

E1pp committed:
YT-22307: Empty template for dist write api
e9554dea04d2a2fb0c9d8cbee012afa88382e715

Commit:
Automatic release build for test_tool, os_ya, ya_bin, os_test_tool

From hash: [8de4949957d3a7517bb29f7ce6af1d4eec61afea](https://a.yandex-team.ru/arcadia/commit/8de4949957d3a7517bb29f7ce6af1d4eec61afea)
From revision: [14696290](https://a.yandex-team.ru/arcadia/commit/rXXXXXX)
[CI flow](https://a.yandex-team.ru/projects/ya_make/ci/releases/flow?dir=devtools%2Fya&id=release-ya-bin2-ya-bin3-tts&version=463)
Flow triggered by user: [robot-ci](https://staff.yandex-team.ru/robot-ci)

Update tools: test_tool, os_ya, ya_bin, os_test_tool
e8fd052c7e045bbfc311a349b246422dbe698ae4

aavdonkin committed:
Added sparsed columns tests for standalone tables (#8167)

vitstn committed:
Removed weird restriction for optional pg (#8175)

Commit:
Temporary enable fix-elf by default
bd9670ad3d9e38289bae7d361cdb0923077bb552

robot-piglet committed:
Intermediate changes

azevaykin committed:
Statistics: retry AnalyzeTable to ColumnShard (#8190)

abyss7 committed:
Wait for all CAs inside Executer before shutdown (#7829)

evanevanevanevannnn committed:
inference json support (#8102)

lll-phill-lll committed:
Try adding flamegraph with simple copy
6d3bfb2f6ddb8ad122ad37485ce2d0ac8346e0ee

alexv-smirnov committed:
build: refresh Embedded UI (v6.16.0) (#8143)

Co-authored-by: astandrik <astandrik@users.noreply.github.com>

azevaykin committed:
Statistics: Remove IsSchemeshardSeen, rely on persisted ScheduleTraversals (#8182)

andrewstalin committed:
Statistics: Expanded ForceTraversalOperations scheme, add CreatedAt (#8194)

alexv-smirnov committed:
Import libraries 240823-1155

rvu1024 committed:
[yt provider] Allow append with same column groups (#8176)

loochek committed:
Fix RebuildCalcOverWindowGroup failing with PercentRank/CumeDist/Ntile (#8148)

molotkov-and committed:
OIDC: Add support redirects (#8151)

siarheivesialou committed:
LOGBROKER 8891 fix parsing queue url (#8203)

swalrus1 committed:
Forbid creating empty tiering rules in column shards (#8154)

alexd65536 committed:
set partitioning settings for statistics table (#8193)

azevaykin committed:
Statistics: reset events in test observers (#8118)

gridnevvvit committed:
subscribe on changes in resource broker (#8187)

MBkkt committed:
Refactor build index (#7658)

robdrynkin committed:
Dsproxy put lwtrack (#8168)

dcherednik committed:
Set EnableUniqConstraint by default (#8180)

ildar-khisambeev committed:
count-min-sketch as index (#7978)

Tony-Romanov committed:
Add stat counters for PutsPerSecond. (#8201)

alexvru committed:
Support STLOG json format for JSON output (#8205)

kungasc committed:
Always limit HardLimitBytes with MemTotal (#8223)

rvu1024 committed:
Handle Write in DQ recapture (#8212)

ivanmorozov333 committed:
dont create temporary sparsed array on merging (#8174)

fedor-miron committed:
Block datetime2 (#6472)

GrigoriyPA committed:
YQ-3570 added s3 wildcards validations (#8017)

Pseudolukian committed:
Rebase

blinkov commented:
Superseded by #8306
